airbyte-webapp/src/views/Connector/RequestConnectorModal/RequestConnectorModal.tsx

```diff
@@ -34,22 +47,120 @@ const RequestConnectorModal: React.FC<RequestConnectorModalProps> = ({ onClose,
     }, 2000);
   };
 
+  const { formatMessage } = useIntl();
+  const dropdownData = [
+    { value: "source", label: <FormattedMessage id="connector.source" /> },
+    {
+      value: "destination",
+      label: <FormattedMessage id="connector.destination" />,
+    },
+  ];
+
   return (
-    <Modal title={<FormattedMessage id="connector.requestConnector" />} onClose={onClose}>
-      <Content>
-        <ConnectorForm
-          onSubmit={onSubmit}
-          onCancel={onClose}
-          currentValues={{
-            connectorType,
-            name: initialName ?? "",
-            additionalInfo: "",
-            email: workspace.email,
-          }}
-          hasFeedback={hasFeedback}
-        />
-      </Content>
-    </Modal>
+    <Formik
+      initialValues={{
+        connectorType: connectorType || "",
+        name: searchedConnectorName || "",
+        additionalInfo: "",
+        email: workspaceEmail || "",
+      }}
+      validateOnBlur
+      validateOnChange
+      validationSchema={requestConnectorValidationSchema}
+      onSubmit={onSubmit}
+    >
+      {({ setFieldValue }) => (
+        <Form>
+          <ModalBody className={styles.modalBody}>
```

Comment:
Why are we using `ModalBody` instead of the `Modal` component?

Reply:
We use `Modal`, it's defined in `ModalService`:
https://github.com/airbytehq/airbyte/blob/7c4980b065633bba44ba15dcb8d0a12f505fc536/airbyte-webapp/src/hooks/services/Modal/ModalService.tsx#L41-L57